index.html

```diff
@@ -9137,6 +9138,24 @@ <h2 id="AudioWorkletNodeOptions">
                   <a data-link-for="AudioNode">numberOfOutputs</a> attribute.
                 </dd>
                 <dt>
+                  <code><dfn>outputChannelCount</dfn></code> of type
+                  <span class="idlAttrType"><code>sequence&lt;unsigned
+                  long&gt;</code></span>
+                </dt>
+                <dd>
+                  This array is used to configure the number of channels in
+                  each output. For example, <code>outputChannelCount: [n,
+                  m]</code> specifies the number of channels in the first
+                  output to <code>n</code> and the second output to
+                  <code>m</code> respectively. <code>IndexSizeError</code> MUST
+                  be thrown if the length of sequence does not match
+                  <a data-link-for=
+                  "AudioWorkletNodeOptions">numberOfOutputs</a>. A
+                  <code>NotSupportedError</code> exception MUST be thrown if a
+                  channel count is not in the valid range of <a data-link-for=
+                  "AudioNode">channelCount</a>.
```

Comment:
This seems confusing -- first of all, `channelCount` is not a range, it's a single number. Also, it's not clear if you mean the option dictionary value `AudioNodeOptions.channelCount`, or the read-only attribute `AudioNode.channelCount` (I'm guessing you mean the options value, but then you have to account for what happens if this was omitted, in which case there is no defined channel count for up/downmixing).

Reply:
What I meant was `AudioNode.channelCount` here. We have a defined range for that property and the same rules should be applied to each value in `AudioWorkletNodeOptions.outputChannelCount` array. I thought `data-link-for` was quite clear, but it's not visible in the actual page. So I edited the text to clarify what `channelCount` means here.

Also the behavior on unspecified (omitted) `outputChannelCount` is described in the following section.